executor/src/lib.rs

```diff
@@ -309,17 +313,25 @@ where
         storage_write_client: Arc<dyn StorageWrite>,
         config: &NodeConfig,
     ) -> Self {
+        let rt = Runtime::new().unwrap();
```

Comment:
Shouldn't we use `.expect("Can't get runtime")` or something instead?

Reply:
I've found that at least for runtime creation there isn't much value added by an additional string since the error you get should be pretty self explanatory.